Airbnb.Application/Options/KafkaOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace Airbnb.Application.Options;
+
+public class KafkaOptions
+{
+    public string BootstrapServers { get; set; } = default!;
+    public string Topic { get; set; } = default!;
```

Comment:
No sense for topic configuration via appsettings because a single app can produce/consume from different topics. Better create class Topics or TopicRegistry. There you can locate all your topics. And moreover you can create generic publisher/consumer to decide from which topic to read/produce by event type